onnxruntime/test/ir/graph_test.cc

```diff
@@ -907,5 +907,66 @@ TEST(NameResolutionTest, DuplicateName) {
   bool duplicate_error_found = status.ErrorMessage().find("Duplicate") != std::string::npos;
   EXPECT_TRUE(duplicate_error_found);
 }
+
+TEST(GraphUpdateTest, ReplaceInitializedTensor) {
+  Model model{"GraphUpdateTest"};
```

Comment:
Would be good to test that Graph::ToGraphProto has the correct value before and after a replacement.